CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Table of Contents
 
- - [2.0.5](#204---TBD)
+ - [2.0.5](#204---20211029)
```

Comment:
The anchor was broken and I didn't look at this til after the end of the EDT day, so bumping this to Monday also.
```suggestion
 - [2.0.5](#205---20211101)
```